spec/helpers/active_admin_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Features
+  def preconditions
+    AdminUser.create(email: 'admin@example.com', password: 'password')
+    Author.create(first_name: 'Janna', last_name: 'Aguzarova')
+    Category.create(title: 'Horror')
+    Book.create(title: 'Wonderland', price: 999, inventory: 1, author_id: 1, category_id: 1)
```

Comment:
Never use fixed ids for tests. Auto increment of primary keys is not predictable. It can be any number in huge test suite.